S/Sundials/Compat.toml

```diff
@@ -79,11 +79,15 @@ julia = "1"
 ["2.5-2"]
 DiffEqBase = "4.29-5"
 
-["2.7-3"]
+["2.7-3.3"]
 DataStructures = "0.15-0"
 
 ["3-3.1"]
 DiffEqBase = "5"
 
-["3.2-3"]
+["3.2-3.3"]
 DiffEqBase = "5.5-5"
+
+["3.4-3"]
+DataStructures = "0.15.0-*"
+DiffEqBase = "5.5.0-*"
```

Comment:
Pretty optimistic bounds

Reply:
Indeed. That's worth exploring later.

Reply:
...

Reply:
This is going to happen quite a lot now that we're taking what people have in their Project files literally, which is why I didn't want to do it previously. We will cap all of these at some point in the future, however, when we start requiring that all bounds be more reasonable.

Reply:
Yes, we will definitely look at it in the future. But right now I just need to keep developing and I can't play around with the package manager this week, so we're just punting on it so we can get work done. Every time we put upper bounds on DiffEq the resolver breaks though, so IMO it's not worth the time to start looking into bounds until that's fixed. We've resorted to using only lower bounds because it's easier to just fix people's errors. However, I would really welcome a contribution which takes a deep dive into adding upper bounds and ensuring that the it all works well with the resolver.

Reply:
You don't really have to think about it, just use
```
[compat]
DiffEqBase = "5.5"
```
knowing that `DiffEqBase v6` will be breaking.

Reply:
But it has almost no chance of breaking this package.